src/components/AddNewBook.js

```diff
@@ -0,0 +1,29 @@
+const AddNewBook = () => (
+  <div className="book-content">
+    <div className="book-items">
+      <div className="bitem">
+        <h3>Lord of the Rings</h3>
+        <button type="button" className="removebtn">Remove</button>
+      </div>
+      <div className="bitem">
+        <h3>Dragons Den</h3>
+        <button type="button" className="removebtn">Remove</button>
+      </div>
+      <div className="bitem">
+        <h3>Bond</h3>
+        <button type="button" className="removebtn">Remove</button>
+      </div>
+    </div>
```

Comment:
- [ ] There should be a **`Book`** component that takes in the **book data for a single book** as a prop and renders its title and author. Kindly extract a single book here into a new component and make it render props instead.
- [ ] The new Book component should also have a remove **button**. This button does not have to work yet. It is merely a placeholder.